project/plugins.sbt

```diff
@@ -4,4 +4,4 @@ addSbtPlugin("com.geirsson"        % "sbt-ci-release" % "1.5.2")
 addSbtPlugin("org.scoverage"       % "sbt-scoverage"  % "1.6.1")
 addSbtPlugin("ch.epfl.scala"       % "sbt-scalafix"   % "0.9.11")
 addSbtPlugin("org.scalameta"       % "sbt-scalafmt"   % "2.3.1")
-addSbtPlugin("com.beautiful-scala" % "sbt-scalastyle" % "1.2.0")
+addSbtPlugin("com.beautiful-scala" % "sbt-scalastyle" % "1.3.0")
```

Comment:
Looks like something's gone wrong here @fthomas. This PR says it's for `com.beautiful-scala:scalastyle:1.3.0`, but scala-steward updated `com.beautiful-scala:sbt-scalastyle` to `1.3.0`, which doesn't exist yet.

Reply:
Thanks for the hint, @mwz. I added this to [list of PRs that need to be investigated](https://github.com/fthomas/scala-steward/wiki/PRs-to-investigate).

Reply:
Thanks for replying, I'll close this for now, but feel free to let me know if you need this reopened to investigate.